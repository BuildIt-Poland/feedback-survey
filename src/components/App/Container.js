import styled from 'styled-components';

const Container = styled.div`
  /* display: grid;
  grid-template-areas: 'header' 'main' 'footer';
  grid-template-columns: 100%;
  grid-template-rows: auto 1fr auto;
  min-height: 100%; */
  display: flex;
  flex-direction: column;
  min-height: 100vh;
`;

export default Container;
