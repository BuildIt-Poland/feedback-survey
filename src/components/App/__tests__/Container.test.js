import React from 'react';
import { create } from 'react-test-renderer';

import Container from '../Container';

describe('COMPONENT - App Container', () => {
  it('renders correctly', () => {
    const component = create(<Container />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
