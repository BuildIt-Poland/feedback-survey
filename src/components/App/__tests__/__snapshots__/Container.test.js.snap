// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - App Container renders correctly 1`] = `
<div
  className="sc-bdVaJa ebpHPz"
/>
`;
